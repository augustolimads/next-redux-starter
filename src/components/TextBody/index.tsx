import * as S from './styles'

const TextBody = () => (
  <S.Wrapper>
    <h1>TextBody</h1>
  </S.Wrapper>
)

export default TextBody
