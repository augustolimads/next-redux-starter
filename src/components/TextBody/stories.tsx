import { Story, Meta } from '@storybook/react/types-6-0'
import TextBody from '.'

export default {
  title: 'TextBody',
  component: TextBody
} as Meta

export const Default: Story = () => <TextBody />
