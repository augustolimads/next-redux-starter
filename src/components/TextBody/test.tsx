import { render, screen } from '@testing-library/react'

import TextBody from '.'

describe('<TextBody />', () => {
  it('should render the heading', () => {
    const { container } = render(<TextBody />)
    expect(
      screen.getByRole('heading', { name: /TextBody/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
