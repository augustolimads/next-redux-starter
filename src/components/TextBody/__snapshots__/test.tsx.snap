// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TextBody /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    TextBody
  </h1>
</main>
`;
